data/entity/npc/animal/sheep/sheep.npcs.toml

```
@@ -9,12 +9,12 @@ clone = "sheep"
 id = 42
 examine = "Freshly sheared."
 
-[sheep_black_feet_shorn]
+[black_sheep_feet_shorn]
```

```suggestion
[sheep_black_feet_shorn]
```

This is a white sheep